web/src/components/navbar/NavMenuItem.tsx

```diff
@@ -30,11 +30,8 @@ export const NavMenuItem = ({
       selected={selected}
       key={key}
     >
-      {icon && <div className="icon-width">{icon}</div>}
-      <div
-        className={`${icon ? "" : "no-icon-offset"} text-wrap truncate-long-business-names-desktop`}
-        data-testid={dataTestid}
-      >
+      <div className="icon-width">{icon ?? ""}</div>
+      <div className={`text-wrap truncate-long-business-names-desktop`} data-testid={dataTestid}>
```

Comment:
Does this also need `truncate-long-business-names-mobile`?

If so, why don't we have a single class `truncate-long-business-name` that uses a media query to change the value of `max-width`?